### PR TITLE
Layout & Scaffolding

### PR DESCRIPTION
**Pull request by ahmedsamir96m**
just finished the layout and scoffolding sectio

### CODE REVIEW

**Review by Nilegfx — CHANGES_REQUESTED** [review]
in order for CI/CD to work smoothly, we need one directory to act as root directory for static files (html, css, js, images .. etc) before this PR, this directory was `/docs`, please move all static files to a directory named `dist` or `build`

**Review by Nilegfx — CHANGES_REQUESTED** [review]
please remove `assets/js` directory. keep the code always clean, and empty file doesn't add any value for now, whoever start doing javascript should know where to create the file and link it in html. keeping dead/unused files/code in your codebase may confuse others

File: index.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        <button class="btn btn-primary" type="submit">subscribe</button>
+      </form>
+    </div>
+
+    <div class="footer-section__credits">
+      footer credits
+    </div>
+  </footer>
+
+  <script src="./assests/js/main.js"></script>
```
Comment (Nilegfx) [inline index.html]: unused file, better to remove it. the less our code base is, the better. ;)

Comment (ahmedsamir96m) [reply index.html]: done!

**Review by Nilegfx — APPROVED** [review]
LGTM

Happy to have this as the very first PR from the mentees, That will be a remarkable moment for all of us isA. 

Good job ya Ahmed!

**Pull request merged**